apps/babanews/pages/_app.tsx

```diff
@@ -120,7 +120,6 @@ function CustomApp({Component, pageProps, emotionCache}: CustomAppProps) {
                 slug="main"
                 headerSlug="header"
                 iconSlug="icons"
-                subscriptionsUrl={null}
```

Comment:
This has been removed everywhere but not replaced. What was this for?

Reply:
Simply doesn't exist anymore